test/e2e/multi-cluster-deployment/setup-kubestellar.sh

```diff
@@ -13,27 +13,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+set -x # echo so users can understand what is happening
 set -e # exit on error
 
 echo "Create a Kind hosting cluster with nginx ingress controller and KubeFlex operator"
 echo "-------------------------------------------------------------------------"
-kflex init --create-kind &> /dev/null
+kflex init --create-kind
```

Comment:
The problem is that kflex is using blocking commands and the github workflow sample that and bombard the output with a lot of messages which make the output unusable. 
We need to differentiate between local usage (where we want to see the output) and cicd test where we want pus to /dev/null.
Maybe add a redirection flag that can be set by the run-test which is not used in cicd

Reply:
This change is from #1572, see the discussion there.

Reply:
@ezrasilvera  we can open an issue in kubeflex to add a flag to the cli so that for ci/cd tests we do not do the noisy updates

Reply:
https://github.com/kubestellar/kubeflex/issues/173